src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -1115,10 +1088,10 @@ public void StateChanged(long index)
                 throw;
             }
         }
-
-        private void NotifyFeaturesAboutStateChange(DatabaseRecord record, long index)
+        
+        private void NotifyFeaturesAboutStateChange(DatabaseRecord record, long index , ClusterStateMachine.DatabaseRecordChange changes = ClusterStateMachine.DatabaseRecordChange.None)
```

Comment:
not sure if having default value here is a good idea, someone will forget about it in the future at we will end up with 'none'